lighthouse/helpers/dart_db.py

```diff
@@ -23,9 +24,16 @@ def get_samples_for_barcode(cnxn, barcode):
         f"SELECT * FROM {app.config['DART_RESULT_VIEW']}"
         f" WHERE [{FIELD_DART_DESTINATION_BARCODE}]='{barcode}'"
         f" AND (([{FIELD_DART_ROOT_SAMPLE_ID}] IS NOT NULL"
+        f" AND [{FIELD_DART_ROOT_SAMPLE_ID}]<>''"
         f" AND [{FIELD_DART_RNA_ID}] IS NOT NULL"
+        f" AND [{FIELD_DART_RNA_ID}]<>''"
         f" AND [{FIELD_DART_LAB_ID}] IS NOT NULL)"
-        f" OR [{FIELD_DART_CONTROL}] IS NOT NULL);"
+        f" AND [{FIELD_DART_LAB_ID}]<>''"
```

Comment:
Should this line be within the bracket above?

Reply:
Yes, sorry